### ISSUE TITLE
Add new loading pages with various css styling and you can also use a loader!!

### ISSUE THREAD

**Issue by Harshita248** [Harshita248/Loading121#1]
_Make open source contribution a fun : )_
**Add the screenshot of the same** so that it becomes easier for others to know which styling has already been used !

**Comment by geeks5551** [comment]
hey can i contribute to the various advanced styling like animation, transition effects , flexbox,media query etc to make the issue more responsive?

**Comment by Harshita248** [comment]
Yes @geeks5551 go ahead !!

**Comment by geeks5551** [comment]
hey I have opened a pull request after creating  the required loading page and loader with a folder named "SHIVAM005" please merge it with main repo , any suggestions/comments  will be highly welcomed

**Comment by Harshita248** [comment]
done @geeks5551 nice work

**Comment by Ritika967** [comment]
Want to work @Harshita248

**Comment by Harshita248** [comment]
Okay @Ritika967  go ahead

**Comment by LeonardoRoccaH** [comment]
Hi can I contribute?

**Comment by Harshita248** [comment]
Yeah @LeonardoRoccaH sure you can make a folder with a relevant name such as loading2 or loader and go ahead!!

**Comment by LeonardoRoccaH** [comment]
Hi @Harshita248  I have created a pull request with a folder named "LOADER" please merge it with main repo, any suggestion or comment is welcome

**Comment by Harshita248** [comment]
okay @LeonardoRoccaH nice
